src/main/scala/com/gd/twitteranalytics/TwitterReportingJob.scala

```diff
@@ -26,13 +27,14 @@ object TwitterReportingJob {
       log.debug("=======> Stage 1: Twitter Data Read Complete !")
       val filteredStatusDf = JsonTranslator.getFilteredDf(spark, tweetsDataFrame)
```

Comment:
'filteredStatusDf' doesn't give a clue what is inside of the Dataframe. What exactly has been filtered in/out?
Please make  the code more self-descriptive.

Reply:
Fixed.